lib/mshoplib/src/MShop/Supplier/Manager/List/Type/Default.php

```diff
@@ -0,0 +1,502 @@
+	private $_searchConfig = array(
+		'supplier.list.type.id' => array(
+			'code'=>'product.list.type.id',
+			'internalcode'=>'msuplity."id"',
+			'internaldeps'=>array( 'LEFT JOIN "mshop_supplier_list_type" AS msuplity ON ( mproli."typeid" = msuplity."id" )' ),
+			'label'=>'Supplier list type ID',
+			'type'=> 'integer',
+			'internaltype' => MW_DB_Statement_Abstract::PARAM_INT,
+			'public' => false,
+		),
+		'supplier.list.type.siteid' => array(
+			'code'=>'product.list.type.siteid',
+			'internalcode'=>'msuplity."siteid"',
+			'label'=>'Supplier list type site ID',
+			'type'=> 'integer',
+			'internaltype' => MW_DB_Statement_Abstract::PARAM_INT,
+			'public' => false,
+		),
+		'supplier.list.type.code' => array(
+			'code'=>'product.list.type.code',
+			'internalcode'=>'msuplity."code"',
+			'label'=>'Supplier list type code',
+			'type'=> 'string',
+			'internaltype' => MW_DB_Statement_Abstract::PARAM_STR,
+		),
+		'supplier.list.type.domain' => array(
+			'code'=>'product.list.type.domain',
+			'internalcode'=>'msuplity."domain"',
+			'label'=>'Supplier list type domain',
+			'type'=> 'string',
+			'internaltype' => MW_DB_Statement_Abstract::PARAM_STR,
+		),
+		'supplier.list.type.label' => array(
+			'code'=>'product.list.type.label',
+			'internalcode'=>'msuplity."label"',
+			'label'=>'Supplier list type label',
+			'type'=> 'string',
+			'internaltype' => MW_DB_Statement_Abstract::PARAM_STR,
+		),
+		'supplier.list.type.status' => array(
+			'code'=>'product.list.type.status',
+			'internalcode'=>'msuplity."status"',
+			'label'=>'Supplier list type status',
+			'type'=> 'integer',
+			'internaltype' => MW_DB_Statement_Abstract::PARAM_INT,
+		),
+		'supplier.list.type.ctime'=> array(
+			'code'=>'product.list.type.ctime',
+			'internalcode'=>'msuplity."ctime"',
+			'label'=>'Supplier list type create date/time',
+			'type'=> 'datetime',
+			'internaltype'=> MW_DB_Statement_Abstract::PARAM_STR,
+		),
+		'supplier.list.type.mtime'=> array(
+			'code'=>'product.list.type.mtime',
+			'internalcode'=>'msuplity."mtime"',
+			'label'=>'Supplier list type modification date/time',
+			'type'=> 'datetime',
+			'internaltype'=> MW_DB_Statement_Abstract::PARAM_STR,
+		),
+		'supplier.list.type.editor'=> array(
+			'code'=>'product.list.type.editor',
+			'internalcode'=>'msuplity."editor"',
+			'label'=>'Supplier list type editor',
+			'type'=> 'string',
+			'internaltype'=> MW_DB_Statement_Abstract::PARAM_STR,
+		),
+	);
+
+
+	/**
+	 * Initializes the object.
+	 *
+	 * @param MShop_Context_Item_Interface $context Context object
+	 */
+	public function __construct( MShop_Context_Item_Interface $context )
+	{
+		parent::__construct( $context );
+		$this->_setResourceName( 'db-supplier' );
+	}
+
+
+	/**
+	 * Removes old entries from the storage.
+	 *
+	 * @param integer[] $siteids List of IDs for sites whose entries should be deleted
+	 */
+	public function cleanup( array $siteids )
+	{
+		$path = 'classes/product/manager/list/type/submanagers';
```

Comment:
Must be "classes/supplier/...". This was a bug in the customer list class which have been already fixed :-)